index.html

```diff
@@ -73,7 +73,7 @@
             companyURL: "https://www.transmute.industries/",
             note: "v2.0", w3cid: 109171},
           { name: "Michael B. Jones", url: "https://self-issued.info/",
-            company: "Microsoft", companyURL: "https://www.microsoft.com",
+            company: "independent",
```

Comment:
Would you like to add your website too? Like in VC-JWT repo PR?

Reply:
I'm happy with having the link to my blog for the moment.  I may add a companyURL in the future when I've developed a Web site for my consulting business, but this is fine for now.